periflow/sdk/resource/checkpoint.py

```diff
@@ -900,12 +901,14 @@ def convert(
                 "Start loading Hugging Face checkpoint(%s) for conversion...",
                 model_name_or_path,
             )
-            state_dict = hf_factory.from_pretrained(
-                model_name_or_path,
-                torch_dtype=torch.float32,
-                cache_dir=cache_dir,
-                trust_remote_code=True,
-            ).state_dict()
+            with accelerate.init_empty_weights():
```

Comment:
Why don't we set `include_buffers=True`?

Reply:
I found that Using context manager's `incldue-buffers=True` option with `from_pretrained(..., device_map='cpu')` doesn't work if we use include_buffers=True. So, I've changed to use `low_cpu_mem_usage=True` for same purpose.